README.md

```diff
@@ -8,7 +8,7 @@ Welcome to the C# extension for Visual Studio Code! This preview provides the fo
 
 * Lightweight development tools for [.NET Core](https://dotnet.github.io).
 * Great C# editing support, including Syntax Highlighting, IntelliSense, Go to Definition, Find All References, etc.
-* Debugging support for .NET Core (CoreCLR). NOTE: Mono and Desktop CLR debugging is not supported.
+* Debugging support for .NET Core (CoreCLR). NOTE: Mono debugging is not support. Desktop CLR debugging has [limited support](https://github.com/OmniSharp/omnisharp-vscode/wiki/Desktop-.NET-Framework).
```

Comment:
support should have an `ed` at the end.

```
debugging is not support**ed**
```

otherwise, looks great to me!

:ship: :it: